pkg/server/commands/listusers/list_users_rpc.go

```diff
@@ -47,6 +48,12 @@ type expandResponse struct {
 	err      error
 }
 
+type foundUser struct {
+	user           *openfgav1.User
+	excludedUsers  []*openfgav1.User
+	noRelationship bool
```

Comment:
can you add godoc explaining what this means?

Reply:
Yep